core/src/toga/window.py

```diff
@@ -135,10 +137,10 @@ def __init__(
         :param id: A unique identifier for the window. If not provided, one will be
             automatically generated.
         :param title: Title for the window. Defaults to "Toga".
-        :param position: Position of the window, as a tuple of ``(x, y)`` coordinates,
-            in :ref:`CSS pixels <css-units>`.
-        :param size: Size of the window, as a tuple of ``(width, height)``, in :ref:`CSS
-            pixels <css-units>`.
+        :param position: Position of the window, as a ``toga.Tuple`` or tuple of
+            ``(x, y)`` coordinates, in :ref:`CSS pixels <css-units>`.
+        :param size: Size of the window, as a ``toga.Size`` or tuple of
+            ``(width, height)``, in :ref:`CSS pixels <css-units>`.
```

Comment:
Should be `toga.Position` rather than `toga.Tuple`. 

Also, wherever either the Position or Size classes appears in the documentation, there should be a hyperlink to their own docs page. This can be done using the `:any:` syntax as shown below.